### PR TITLE
Applied spotless formatter

### PR DESCRIPTION
**Pull request by cweitkamp**
- Applied spotless formatter

After #1294 the ordering of imports were mixed up. I applied the spotless `mvn spotless:apply` command on the whole repo.

Signed-off-by: Christoph Weitkamp <github@christophweitkamp.de>

### CODE REVIEW

File: features/karaf/openhab-tp/src/main/feature/feature.xml [hunk 1]
```diff
@@ -1,310 +1,300 @@
...
-	information.
-
-	This program and the accompanying materials are made available under the
-	terms of the Eclipse Public License 2.0 which is available at
-	http://www.eclipse.org/legal/epl-2.0
-
-	SPDX-License-Identifier: EPL-2.0
-
--->
+<!-- Copyright (c) 2010-2019 Contributors to the openHAB project See the NOTICE file(s) distributed with this work for additional
```
Comment (wborn) [inline features/karaf/openhab-tp/src/main/feature/feature.xml]: It shouldn't format these XML headers.

Comment (cweitkamp) [reply features/karaf/openhab-tp/src/main/feature/feature.xml]: I fixed the headers by running `mvn license:format`.

@seime Do you have an idea how to exclude these parts from automatical formatting?

Comment (seime) [reply features/karaf/openhab-tp/src/main/feature/feature.xml]: @cweitkamp 
There is a generic option avoid joining comments that could be added to the xml formatter settings file
```
formatCommentJoinLines=false
```

Comment (cweitkamp) [reply features/karaf/openhab-tp/src/main/feature/feature.xml]: Thanks. I created https://github.com/openhab/static-code-analysis/pull/375 for it.

**Pull request merged**